modules/ROOT/examples/live-demos/custom-toolbar-menu-button-search/index.js

```diff
@@ -0,0 +1,53 @@
+tinymce.init({
```

Comment:
I wasn't intending to do an example, but I felt that my explanation of how it worked was pretty hard to follow, and an example would help. Plus with this livedemo system that has been setup, it wasn't as painful as I was expecting.

Reply:
I'm not sure what our rules are around having multiple demos on the same page, but I believe the folder this demo is within should reflect the name of the page it is included on. Perhaps the `custom-menu-toolbar-button` page could be split into two pages, and as reflected in the nav by a heading with two sub pages, one for the simple menu button and one for the searchable menu button? This would be a question for @brian-forte .

Reply:
We have multiple demos on the one page where it makes sense to do it like this. We do have the primary demo try and match the page name so it's easier to correlate where it makes sense to though. For example: https://github.com/tinymce/tinymce-docs/blob/e6829425aec9f3f5f54e6b4eb7d426a7ebc6de1f/modules/ROOT/pages/autocompleter.adoc#L195-L201

Reply:
Ah okay, thanks for clarifying.